torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2312,6 +2312,39 @@ def _call_function(
         )
 
 
+class PrintHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def _call_function(
+        self,
+        tx: "InstructionTranslator",
+        args: "list[VariableTracker]",
+        kwargs: "dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+
+        if not isinstance(args[0], str):
+            unimplemented(
```

Comment:
I think we should just error? even if we graph break, the implementation of `hoo.print` would error since the input is not a string